Project.toml

```diff
@@ -1,32 +1,32 @@
 name = "Weave"
 uuid = "44d3d7a6-8a23-5bf8-98c5-b353f8df5ec9"
-version="0.9.0"
+version = "0.9.0"
 
 [deps]
-Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
-Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
-Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Base64 = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
-REPL = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
 Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
-Serialization = "9e88b42a-f829-5b0c-bbe9-9e923198166b"
-JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
+Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Highlights = "eafb193a-b7ab-5a9e-9068-77385905fa72"
+JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
+Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
 Mustache = "ffc61752-8dc7-55ee-8c37-f3e9cdd09e70"
-YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+REPL = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
+Serialization = "9e88b42a-f829-5b0c-bbe9-9e923198166b"
+YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
 
 [compat]
 Highlights = ">=0.3.1"
+Mustache = ">=0.4.1"
 Plots = ">=0.19.0"
 YAML = ">=0.3.0"
-Mustache = ">=0.4.1"
 
 [extras]
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
-Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
-Gadfly = "c91e804a-d5a3-530f-b6f0-dfbca275c004"
 Cairo = "159f3aea-2a34-519c-b102-8c37f9878175"
+Gadfly = "c91e804a-d5a3-530f-b6f0-dfbca275c004"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [targets]
 test = ["Test", "Plots", "Gadfly", "Cairo"]
```

Comment:
You should probably add `Conda` and `IJulia` to `[extras]` and to test under `[targets]` for the installation to work on Travis

Reply:
Sure, will do.